client/src/components/ContactForm.jsx

```diff
@@ -32,7 +32,9 @@ function ContactForm() {
         placeholder="Your message ..."
         required
       />
-      <button type="submit">Submit</button>
+      <button type="submit" aria-label="submit">
```

Comment:
Idk if the aria label is mandatory here with the "submit" juste beneath. Probably dead code

Reply:
Same on all button